satellite/satellitedb/projectmembers.go

```diff
@@ -32,16 +32,52 @@ func (pm *projectMembers) GetByMemberID(ctx context.Context, memberID uuid.UUID)
 }
 
 // GetByProjectID is a method for querying project members from the database by projectID, offset and limit.
-func (pm *projectMembers) GetByProjectID(ctx context.Context, projectID uuid.UUID, pagination console.Pagination) (_ []console.ProjectMember, err error) {
+func (pm *projectMembers) GetPagedByProjectID(ctx context.Context, projectID uuid.UUID, cursor console.ProjectMembersCursor) (_ *console.ProjectMembersPage, err error) {
 	defer mon.Task()(&ctx)(&err)
-	if pagination.Limit < 0 || pagination.Offset < 0 {
-		return nil, errs.New("invalid pagination argument")
+
+	search := "%" + strings.Replace(cursor.Search, " ", "%", -1) + "%"
+
+	if cursor.Limit > 50 {
+		cursor.Limit = 50
 	}
 
-	var projectMembers []console.ProjectMember
-	searchSubQuery := "%" + strings.Replace(pagination.Search, " ", "%", -1) + "%"
-	//`+getOrder(pagination.Order)+`
+	if cursor.Page == 0 {
+		return nil, errs.New("page cannot be 0")
+	}
+
+	page := &console.ProjectMembersPage{
+		Search: cursor.Search,
+		Limit:  cursor.Limit,
+		Offset: uint64((cursor.Page - 1) * cursor.Limit),
+	}
 
+	countQuery := pm.db.Rebind(`
```

Comment:
```
	countQuery := pm.db.Rebind(`
		SELECT COUNT(*)
		FROM project_members
		INNER JOIN users ON project_members.member_id = u.id
		WHERE project_members.project_id = ?
			AND ( users.email LIKE ? OR 
				  users.full_name LIKE ? OR
				  users.short_name LIKE ? 
			)`)
```

The `WHERE` belongs to the main `SELECT` so might make more sense to remove the identation.